modules/ROOT/pages/dw-core-functions-plusplus.adoc

```diff
@@ -207,7 +207,7 @@ output application/json
 
 
 [[plusplus5]]
-== &#43;&#43;&#40;LocalTime, Date&#41;: LocalDateTime
+== &#43;&#43;&#40;time: LocalTime, date: Date&#41;: LocalDateTime
```

Comment:
TODO: localtime

Reply:
also make sure that the param names in the param tables (not shown) are listed in the correct order